docs/applications/bitcoin-core.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+title: Bitcoin Core
+parent: Main Applications
+nav_order: 610
+---
+## Bitcoin Core
+
+Bitcoin Core the most popular software implementation of Bitcoin, openly developed on GitHub (<https://github.com/bitcoin/bitcoin>) and publishing releases on <https://bitcoincore.org>. The BitBox base uses the latest stable binary release to communicate with the Bitcoin peer-to-peer network, for example to learn about current transactions and receive newly mined blocks. All data is regarded as untrusted until it is locally verified to the consensus rules enforced by the BitBox Base.
+
+### Using binary releases
+
+While the `master` branch on GitHub reflects the current development process, a new stable release is tagged and tested every few months. This release is built from the source code in a reproduciable way for various platforms by many contributors, verifying that they get the exact same binary executables by comparing the sha256 hash. These hash values for all images are provided in the `SHA256SUM.asc` file, which is signed by a Bitcoin Core maintainer, currently [laanwj](https://github.com/laanwj).
+
+Instead of compiling Bitcoin Core from source when building the Armbian image, or poviding our own binaries, we believe that following the official binary releases is a good way to minimize trusting any single entity and keep the build process performant.
+
+To ensure that the official binary is used, we store the verified signing key `90C8 019E 36C2 E964` independently from the Bitcoin Core release site and validate the downloaded binary against the signed hash values. If this verification does not succeed (a single bit difference would be enough) the build script abort with an error.
+
+### Configuration
+
+The application configuration is specified in the local `/etc/bitcoin/bitcoin.conf` file. Please check the most current initial configuration in [`customize-armbian-rockpro64.sh`](../../armbian/base/build/customize-armbian-rockpro64.sh).
+
+```
+# application
+testnet=1
+server=1
+listen=1
+daemon=1
+txindex=0
+prune=0
+disablewallet=1
+pid=/run/bitcoind/bitcoind.pid
+rpccookiefile=/mnt/ssd/bitcoin/.bitcoin/.cookie
+sysparms=1
+rpcconnect=127.0.0.1
+
+# performance
+dbcache=2000
+maxmempool=50
+maxconnections=40
+maxuploadtarget=5000
+
+# tor
+proxy=127.0.0.1:9050
+seednode=nkf5e6b7pl4jfd4a.onion
+seednode=xqzfakpeuvrobvpj.onion
+seednode=tsyvzsqwa2kkf6b2.onion
+```
+
+Some notes about this specific configuration:
+
+* **Application options**
```

Comment:
Similar to my comment above, ideally we'd find a way to put this (excellent!) documentation around the different `bitcoin.conf` options very near the actual config. Perhaps it would be possible to add this information directly inline in the `bitcoin.conf` file, and then inline it inside this `.md` file? We could do this in a follow-up PR, but wanted to raise the idea.

Reply:
Agreed.